app/src/main/java/com/infomaniak/drive/ui/LaunchActivity.kt

```diff
@@ -184,7 +188,10 @@ class LaunchActivity : AppCompatActivity() {
     }
 
     private fun handleShortcuts() {
-        intent.extras?.getString(SHORTCUTS_TAG)?.let { extrasMainActivity = MainActivityArgs(shortcutId = it).toBundle() }
+        intent.extras?.getString(SHORTCUTS_TAG)?.let {
```

Comment:
You can add a name for `it`

Reply:
And you can remove the argument name then